go.mod

```diff
@@ -5,9 +5,9 @@ replace dagger.io/dagger => ./sdk/go
 go 1.18
 
 require (
-	dagger.io/dagger v0.3.0-alpha.4
```

Comment:
Did this happen automatically?

Reply:
Yeah, which is a bit spooky. I'm not sure what this will mean for the release. I decided to commit it and call it out since we're gonna run into it someday, better to see what happens.

Reply:
So it gets removed when I do `go get github.com/moby/buildkit@master`.

And then when I `go get dagger.io/dagger@v0.3.0-alpha.4` to bring it back, it un-bumps `github.com/moby/buildkit` to `v0.10.5`. :rage: 

Probably because `dagger.io/dagger` ends up depending on `github.com/moby/buildkit` and Go actually sees `v0.10.1-0.20221027014600-b78713cdd127` => `v0.10.5` as a version bump. :sob:

Reply:
> The fix here seems to be to add this line instead:

If we do that, then I think our SDK users will also need to include that replace, right? 😢

Reply:
Yeah, I just ran into that. :man_facepalming:

Reply:
It sucks, but right now I'm leaning towards just doing it since it's kind of a drop in the bucket, and it'll go away at the same time that we switch to an external engine.

Even without that `replace` line we'll need to update the `docker/docker` replace, so they'll already need to do some more copy-pasta.

The alternative of course is to go back to vendoring the filesync implementation.